esp/src/src-react/components/Metrics.tsx

```diff
@@ -28,6 +28,82 @@ const defaultUIState = {
     hasSelection: false
 };
 
+interface PropertyValue {
+    Key: string;
+    Value: string;
+    Avg: string;
+    Min: string;
+    Max: string;
+    Delta: string;
+    StdDev: string;
+    SkewMin: string;
+    SkewMax: string;
+    NodeMin: string;
+    NodeMax: string;
+}
+
+const regex = /[A-Z][a-z]*/g;
+function splitKey(key: string): [string, string, string] | null {
+
+    // Related properties  ---
+    const relatedProps = ["SkewMin", "SkewMax", "NodeMin", "NodeMax"];
```

Comment:
These were meant to follow a consistent format i.e.

`<Type> <Modifier> <Property>
`

Where the type reflects the underlying measure and the combination of modifer and property were unique.  However I see with min and max I failed to keep that distinction.  I think it probably should have been SkewMinSkewX etc., but almost certainly too late to change now.